README.md

```diff
@@ -42,23 +42,25 @@ and the value is the string you want to garble. An example file is below:
 The values strings are read by `ssg`, garbled, and the [resulting output](https://github.com/NormDoximity/swift-string-garbler/blob/main/Tests/ssgTests/ProjectKeys.swift) is a swift source file you
 can include in your app project. `ssg` will use the values specified in the environment file, unless a
 corresponding key is found as an environment variable in the run time environment of the executing process.
-Should an environment variable be found, that value will be used instead of the value in the passed in 
+Should an environment variable be found, that value will be used instead of the value in the passed in
 file.
 
-Although you can use `ssg` in an ad-hoc capacity, ideally you will want to use this as a build phase in 
+Although you can use `ssg` in an ad-hoc capacity, ideally you will want to use this as a build phase in
 your project. To avoid having to rebuild the source each time you run `ssg` and there are no environment
-file changes, `ssg` can create a checkusm of the key values pairs found in it's environment. Providing 
-a filesname with `--checksum-path` option will read the checksum from a previous run and compare it 
-to the new checksum computed from the environment. Should they differ, the new checksum is written at 
+file changes, `ssg` can create a checkusm of the key values pairs found in it's environment. Providing
+a filesname with `--checksum-path` option will read the checksum from a previous run and compare it
+to the new checksum computed from the environment. Should they differ, the new checksum is written at
 the checksum path and a new swift source file is created.
 
 ### Leaving Comments
-`ssg` supports comments in the JSON file if they have the prefix of `__COMMENT__`. When `ssg` encounters these, those entires are just skipped.
+In addtion to `//` line comments, `ssg` supports comments in the JSON file if they have the prefix of `__COMMENT__`. When `ssg` encounters these, those entires are just skipped.
```

Comment:
I was reluctant to remove it as I wasn't sure that the Dox app wasn't using that commenting style - I thought it might be since you added that feature. Otherwise, I think we can remove it for our purposes. I chose json as the input format largely because it's a one-liner to slurp it up with no outside dependencies. We could just as well use a .ini style file format - the configuration data is more important than the format of that data, at least that's how i look at it.  Philosophy aside, I'm happy to remove the `__COMMENT__` feature if you'd rather use `//` going forward.